README.md

```diff
@@ -145,7 +145,7 @@ See [CONTRIBUTING.md](CONTRIBUTING.md)
 Approvers ([@open-telemetry/java-approvers](https://github.com/orgs/open-telemetry/teams/java-approvers)):
 
 - [Armin Ruech](https://github.com/arminru), Dynatrace
-- [Pavol Loffay](https://github.com/pavolloffay), RedHat
+- [Pavol Loffay](https://github.com/pavolloffay), Traceable AI
```

Comment:
@pavolloffay isn't it just "Traceable" ?

Reply:
I went off his LinkedIn, if that's wrong I can adjust

Reply:
if that's what it says on LinkedIn, then it's probably correct. :) I just looked at their website, and it just says Traceable everywhere, which is what I based my comment on. :)

Reply:
https://github.com/open-telemetry/opentelemetry-java/pull/1862#pullrequestreview-515678775 --> 4fee432